core/tests/protractor_utils/SubTopicViewerPage.js

```diff
@@ -39,6 +51,31 @@ var SubTopicViewerPage = function() {
       expect(await subTopicTileList.count()).toEqual(count);
     }
   };
+
+  this.expectedConceptCardCountToBe = async function(count) {
```

Comment:
Can you make this present tense? For example:

```suggestion
  this.expectConceptCardCountToBe = async function(count) {
```

Reply:
Done